cognite_toolkit/examples/cdf_oid_example_data/raw/workitem2assets.yaml

```diff
@@ -1,2 +1,2 @@
-dbName: {{raw_db}}
+dbName: {{raw_db_workorder}}
```

Comment:
Note here that the user will have full flexibility in naming all the databases, so they can be named all over the place. What we did with e.g. infield was to hardcode prefixes/suffixes across entities where we wanted consitency and use a variable for a subset. I.e. 
`dbName: fixedPrefix_{{variable}}_suffix`

This way you enforce a consistency across all databases in this module.